### PR TITLE
add migration to create and configure staging schema

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/mechanic/upgrades/CreateStagingSchemaForExistingOrgs.kt [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                        ))
+                    }
+
+                    // update search path for server user to include staging schema
+                    stmt.execute(setSearchPathSql(
+                            assemblerConfiguration.server["username"].toString(),
+                            false,
+                            AssemblerConnectionManager.INTEGRATIONS_SCHEMA,
+                            AssemblerConnectionManager.MATERIALIZED_VIEWS_SCHEMA,
+                            AssemblerConnectionManager.PUBLIC_SCHEMA,
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/mechanic/upgrades/CreateStagingSchemaForExistingOrgs.kt:75]: What do we have in the public schema?

**Review by geekbeast — APPROVED** [review]
LOOKS SO GOOD.

**Review by makosblade — APPROVED** [review]
🔥 🎉

File: src/main/kotlin/com/openlattice/mechanic/upgrades/CreateStagingSchemaForExistingOrgs.kt (hunk 1 above)
Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/mechanic/upgrades/CreateStagingSchemaForExistingOrgs.kt:75]: no idea -- this code roughly exists right now in AssemblerConnectionManager (minus the staging schema) on org creation so i just took this list from there. We do also revoke all permissions for user accounts on the public schema -- maybe this was somehow tied to old MV code?

**Pull request merged**